infra/azure.tf

```diff
@@ -145,10 +145,31 @@ CRON
 
 chmod +x /root/daily-reset.sh
 
+cat <<GET_TARGETS > /root/get-targets.sh
+#!/usr/bin/env bash
+set -euo pipefail
+
+arg_target="$1"
```

Comment:
This line currently fails at machine startup - I think you need either a backslash here, or single quotes around `GET_TARGETS`.

Reply:
oh so that's why `'CRON'` is quoted, TIL :)

Reply:
fixed it now, thank you!

Reply:
I'm not seeing the fix - on `main` right now, `GET_TARGET` is not quoted and `$1` is not escaped. Did you perhaps forget to push?

I'll make a PR to correct.